thirdparty/nginx/zimbra-nginx/rpm/SPECS/nginx.spec

```diff
@@ -17,6 +17,8 @@ URL:                http://nginx.org
 The Zimbra nginx build
 
 %changelog
+* Wed Apr 18 2018  Zimbra Packaging Services <packaging-devel@zimbra.com> - VERSION-1zimbra8.7b8ZAPPEND
+- ZCS-4021 for JWT support.
```

Comment:
Is this the proper version to be setting this to?  Do we version the nginx updates independently from the version of ZCS they ship with?

Reply:
Yeah @tmclane  i checked with @Prashantsurana  and the version is fine.

Reply:
Thanks for checking!